docker/Dockerfile

```diff
@@ -47,7 +47,7 @@ COPY --chown=trino:trino gateway-ha /usr/lib/trino
 
 EXPOSE 8080
 USER trino:trino
-CMD java --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED -jar /usr/lib/trino/gateway-ha-jar-with-dependencies.jar "server" "/opt/trino/gateway-ha-config.yml"
+CMD java --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED -jar /usr/lib/trino/gateway-ha-jar-with-dependencies.jar "/opt/trino/gateway-ha-config.yml"
```

Comment:
Can you also update
https://github.com/trinodb/trino-gateway/blob/main/docs/development.md?plain=1#L35 too?